src/ucp/api/ucp.h

```diff
@@ -2100,6 +2137,35 @@ ucs_status_t ucp_context_query(ucp_context_h context_p,
                                ucp_context_attr_t *attr);
 
 
+/**
+ * @ingroup UCP_MEM
+ * @brief Compare two remote keys according to parameters
+ *
+ * This routine returns equal if all the transports can compare equal and
+ * configurations are equal. At least one compare mode is needed.
+ *
+ * The comparison result is < 0 if rkey1 is lower than rkey2, 0 if they
+ * are equal or > 0 if rkey1 is greater than rkey2.
```

Comment:
The comparison result is < 0 if rkey1 is lower than rkey2, 0 if they
are equal or > 0 if rkey1 is greater than rkey2. The remote key greater-than
semantic is arbitrary and implementation dependent, nevertheless it's guaranteed
to be consistent (transitive and antisymmetric). It can be used, for example, to 
perform an efficient binary search in a sorted array of remote keys.

Reply:
fixed, will use similar message in corresponding uct api header